concordium-consensus/src/Concordium/GlobalState/Persistent/Trie.hs

```diff
@@ -111,63 +186,67 @@ instance Bifunctor (TrieF k) where
 instance (Serialize r, Serialize (Nullable r), Serialize v) => Serialize (TrieF k v r) where
     put (Branch vec) = do
         putWord8 1
-        forM_ vec put
+        forM_ (branchesToList vec) put
```

Comment:
Please provide documentation why we serialize the trie with branches in this representation, and not in the list of (k, v) pairs.